internal/commands/push/command.go

```diff
@@ -280,6 +303,41 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewTextLog("Installed dependencies"))
 	}
 
+	if cmd.inputs.IncludeArchivedDependencies {
+		installDependencies := func() error {
+			s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+			s.Suffix = " Installing dependencies: starting..."
+
+			s.Start()
+			defer s.Stop()
+
+			if err := clients.Realm.ImportDependencies(appRemote.GroupID, appRemote.AppID, uploadPathDependencies); err != nil {
+				return err
+			}
+
+			status := realm.DependenciesStatus{State: realm.DependenciesStateCreated}
+			for status.State == realm.DependenciesStateCreated {
+				var err error
```

Comment:
[nit] I don't think we need this err declaration given that `err` is defined in the upper scope